explorer-ui/src/components/NavBar/NavBar.jsx

```diff
@@ -1,18 +1,62 @@
-import * as React from "react"
-import "./NavBar.css"
-
+import * as React from "react";
+import "./NavBar.css";
+import Logo from "../Home/Logo/Logo";
+import { Link } from 'react-router-dom';
 export default function NavBar({ isLoggedIn, handleLogout }) {
-    const onClick = event => {
-        event.preventDefault();
-        handleLogout()
-    }
+  const onClick = (event) => {
+    event.preventDefault();
+    handleLogout();
+  };
 
-    return (
-        <div id="NavBar">
-            <span>Explorer</span>
-            {isLoggedIn &&
-                <a href="#" onClick={onClick}>Logout</a>
-            }
+  return (
+    <div id="NavBar">
+      <div className="content">
+        <div className="navbar-logo">
+          <Logo />
+          {isLoggedIn && (
+            <a href="#" onClick={onClick}>
+              Logout
+            </a>
+          )}
+        </div>
+        <ul className="links">
+          <li>
+            <a className="pages" href="">
+              Products
+            </a>
+          </li>
+          <li>
+            <a className="pages" href="#Buy">
+              Learn
+            </a>
+          </li>
+          <li>
+            <a className="pages" href="#About">
+              Safety
+            </a>
+          </li>
+          <li>
+            <a className="pages" href="#Contact">
+              Support
+            </a>
+          </li>
+          <li>
+            <a className="pages" href="#Contact">
```

Comment:
nice are these to be anchors ? i.e scroll to X -> or meant to navigate to Y page ?

Reply:
these don't really serve a purpose. there aren't any pages attached to them, i just added them for texture and to make the page look more like a real web app lol